todoman/ui.py

```diff
@@ -258,41 +283,39 @@ def detailed(self, todo):
             rv = "{}\n\n{}".format(rv, todo.description)
         return rv
 
-    def format_date(self, date):
-        """
-        Returns date in the following format:
-        * if date == today or tomorrow: "Today" or "Tomorrow"
-        * else: return a string representing that date
-        * if no date is supplied, it returns empty_date
-
-        :param datetime.datetime date: a datetime object
-        """
-        if date:
-            assert isinstance(date, datetime)
-            if date.date() in self.special_dates:
-                rv = self.special_dates[date.date()]
-            else:
-                rv = date.strftime(self.date_format)
-            return rv
+    def format_datetime(self, dt):
+        dt = dt or None
+        if dt in self.special_datetimes:
+            format_string = self.special_datetimes[dt]
+        elif dt.date() in self.special_datetimes:
+            format_string = self.special_datetimes[dt.date()]
```

Comment:
Oh, I completely misread this, yes, it makes sense.

I don't think the first bit of the `if` is neccessary though, since we never format mere `date`s.